### DISCUSSION TITLE
Markisensteuerung

### DISCUSSION THREAD

**Discussion by gerd2005** [thkl/hap-homematic#464] [Q&A]
Hallo, ich möchte unseren Markisenmotor ebenfalls in die Homematic und in das Plugin bringen. 
Für alle Rolläden habe ich bspw. den HMIP-FROLL installiert. Mit dem Siri Befehl „öffne/schließe das Rollo XY“ wird  die entsprechende Aktion ausgeführt. 
Wie ist das denn jetzt mit einer Markise, wenn ich hier auch einen FROLL einbaue? Der Befehl öffnen & schließen passt ja nicht 100%ig. Und wie ist das wenn ich sage: „Öffne alle Rollos“. Dann fährt doch sicherlich auch die Markise heraus, oder?

Hat jemand vlt. ein Nest-Practice-Beispiel? 
danke und lieben Gruß.

**Comment by wkettler1985** [comment]
Ich habe genau dieses Problem!
Wenn ich sage Öffne alle Rollos werden alle Rollos geöffnet, aber eben auch die Markise.
Gibt es dafür eine Lösung?

**Reply by ChrWi** [reply]
Basierend auf den identischen Geräten (bei mir die BROLLs) sehe ich hier keine Möglichkeit.
